README.md

```diff
@@ -197,6 +197,43 @@ struct MyView: View {
 }
 ```
 
+### Integration with Spezi Modules
+
+A Spezi [`Module`](https://swiftpackageindex.com/stanfordspezi/spezi/documentation/spezi/module) is a great way of structuring your application into
+different subsystems and provides extensive capabilities to model relationship and dependence between modules.
+Every [`BluetoothDevice`](https://swiftpackageindex.com/stanfordspezi/spezibluetooth/documentation/spezibluetooth/bluetoothdevice) is a `Module`.
+Therefore, you can easily access your SpeziBluetooth device from within any Spezi `Module` using the standard
+[Module Dependency](https://swiftpackageindex.com/stanfordspezi/spezi/documentation/spezi/module-dependency) infrastructure. At the same time,
+every `BluetoothDevice` can benefit from the same capabilities as every other Spezi `Module`.
+
+Below is a short code example demonstrating how a `BluetoothDevice` uses the `@Dependency` property to interact with a Spezi Module that is
+configured within the Spezi application.
+
+```swift
+class Measurements: Module, EnvironmentAccessible, DefaultInitializable {
+    required init() {}
+
+    func recordNewMeasurement(_ measurement: WeightMeasurement) {
+        // ... process measurement
+    }
+}
+
+class MyDevice: BluetoothDevice {
```

Comment:
Amazing; this is such a great mechanism. Great to see how this turned out based on the discussion in https://github.com/StanfordBDHG/ENGAGE-HF-iOS/pull/6 🚀